index.js

```diff
@@ -675,6 +675,34 @@ Client.prototype.publisherInfo = function (publisher, callback) {
   })
 }
 
+// batched interface... note multiple invocations of callback!
+
+Client.prototype.publishersInfo = function (publishers, callback) {
+  if (this.options.version === 'v1') return
+
+  // initial version is still serialized, future versions will use a new API call.
+
+  if (!Array.isArray(publishers)) publishers = [ publishers ]
+  if (publishers.length === 0) return
+
+  this.publishers = this.publishers ? this.publishers.concat(publishers) : underscore.clone(publishers)
+  this._publishersInfo(callback)
```

Comment:
as discussed, we should guard this call if the intent of having the publishers list as an instance property instead of a function argument is to prevent multiple calls in flight.

Reply:
good catch!